pda.sh

```diff
@@ -0,0 +1,42 @@
+package: PDA
+version: "%(tag_basename)s"
+tag: 11.0.7
+source: https://github.com/PascalBoeschoten/pda
```

Comment:
Why not moving this to AliceO2 or alisw? One should use the devel package support of alibuild for development.